test/commands/ProjectCommand.test.ts

```diff
@@ -709,8 +709,8 @@ describe('ProjectCommands', () => {
 
     it('Method getTruffleUnboxCommand should return a value', async () => {
       // Arrange
-      const displayName = 'drizzle';
-      const repoName = 'drizzle-box';
+      const displayName = 'pet-shop';
+      const repoName = 'pet-shop-box';
```

Comment:
I've seen this change in another PR https://github.com/trufflesuite/vscode-ext/pull/231/files#diff-48bfccb76a1533fb09070b496d1aa27c517b3a9c9a10deeb513520fb8efc8607. Just to make sure that this does not cause merge conflicts.

Reply:
Yeah, i had to change it because someone removes Drizzle from the boxes list. So, the tests are breaking all the time.

It won't cause merge conflicts

Thank you again